src/main/java/com/dev/cinema/dao/impl/UserDaoImpl.java

```diff
@@ -12,6 +13,8 @@
 
 @Repository
 public class UserDaoImpl extends AbstractDao<User> implements UserDao {
+    private static final Logger logger = Logger.getLogger(UserDaoImpl.class);
```

Comment:
since you are using lombok let's use it here too

Reply:
agree